scipy/optimize/_highs/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
This and all other licenses must be added to `LICENSES_bundled.txt` in the root of the repo

Reply:
Added to `LICENSES_bundled.txt`